package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "eslint-plugin-khan",
-  "version": "0.0.1",
-  "main": "index.js",
+  "name": "@khanacademy/eslint-plugin-khan",
```

Comment:
We want to namespace all of our packages right?  In this case it's a little redundant though.